### ISSUE TITLE
setPanelFlyoutMenu doesn't work in Audtiion.

### ISSUE THREAD

**Issue by apiscode** [Adobe-CEP/CEP-Resources#394]
AUDT [14.4.0], CEP [11.1.0], Chrome [88], locale: en_US, ui locale: en_US
MacOs BigSur 11.5.2

**Comment by sujaisivanandan** [comment]
HI @apiscode, Have you tried HTML Flyout menu? 
refer example: https://github.com/Adobe-CEP/CEP-Resources/blob/master/CEP_11.x/Samples/CEP_HTML_Test_Extension-10.0/html/HTMLFeatures.html

**Comment by apiscode** [comment]
Yes. I have tried that in my plugin and also in the test extension provided by Adobe.
Have you tried that?

**Comment by sujaisivanandan** [comment]
There are 2 flyout menu in the test extension.
1) the one under "CEP Features"
2) the one under "HTML1"

The flyout menu under HTML1 does work for me with CEP 11

**Comment by apiscode** [comment]
Hi sujaisivanandan

<img width="1295" alt="Screenshot 2021-09-10 at 11 37 41" src="https://user-images.githubusercontent.com/70690497/132834118-a7773880-9720-48ae-8d66-77a1dba17722.png">

OSX version 11.5.2

**Comment by sujaisivanandan** [comment]
@apiscode confirmed that this is not working with Audition 14.4, but works fine in other apps like Illustrator 25.4. Investigating

**Comment by apiscode** [comment]
Any news on that issue? I'm getting every day new updates for Audition Beta.
